lib/src/input/figma/entities/layers/group.dart

```diff
@@ -30,32 +33,36 @@ class Group extends FigmaFrame implements AbstractFigmaNodeFactory {
   @override
   String imageReference;
 
-  Group(
-      {name,
-      isVisible,
-      type,
-      pluginData,
-      sharedPluginData,
-      boundaryRectangle,
-      style,
-      fills,
-      strokes,
-      strokeWeight,
-      strokeAlign,
-      cornerRadius,
-      FigmaConstraints constraints,
-      layoutAlign,
-      size,
-      horizontalPadding,
-      verticalPadding,
-      itemSpacing,
-      List<FigmaNode> children,
-      String UUID,
-      FigmaColor backgroundColor,
-      String transitionNodeID,
-      num transitionDuration,
-      String transitionEasing})
-      : super(
+  @override
+  @JsonKey(ignore: false)
+  FigmaStyleProperty figmaStyleProperty;
+
+  Group({
+    name,
+    isVisible,
+    type,
+    pluginData,
+    sharedPluginData,
+    boundaryRectangle,
+    style,
```

Comment:
Do we need to remove some properties here like `style`, `stroke`, etc.?

Reply:
Done, some of them